src/reducers/MessageCenter.ts

```diff
@@ -29,14 +30,16 @@ export const INITIAL_MESSAGE_CENTER_STATE: MessageCenterState = {
   expandedGroupId: 'default'
 };
 
-const createMessage = (id: number, content: string, type: MessageType) => {
+const createMessage = (id: number, content: string, type: MessageType, count, created, firstTriggered) => {
```

Comment:
No types on the new params.
This is private, so I guess we can live without the types. Just commenting to make sure that you are OK.

Reply:
Oh you are right, I should have types there. Thanks.